Tella/Scenes/Settings/Models/ServerActionType.swift

```diff
@@ -11,14 +11,37 @@ enum ServerActionType: ActionType {
     case delete
 }
 
+enum TemplateActionType: ActionType {
+    case delete
+}
+enum DownloadedTemplateActionType: ActionType {
+    case delete
+    case createEntity
+}
+
 
 var serverActionItems : [ListActionSheetItem] { return [
     
     ListActionSheetItem(imageName: "edit-icon",
-                        content: "Edit",
+                        content: LocalizableUwazi.uwaziServerEdit.localized,
```

Comment:
Thanks for the suggestion! In this case, this looks like a common sheet item, so I think creating a new ActionItem just for this would be an overkill, especially because we are not using any additional data. Only `imageName`, `content`, and `type`. Also, this `ListActionSheetItem` class is being used on all the codebase so creating a new one will be kind of inconsistent

Reply:
Thanks, It's just for organisation and as I told you in the begining of the project I asked about sharing common string between views, I was answered that it's better to have each screen his own strings.